README.md

```diff
@@ -108,14 +108,28 @@ make check
 
 ## Comparison of developer environments
 
+For end-to-end client testing on Qubes, make sure you either run the packaged client or run the unpackaged clent with proxy turned on.
+
 ### Developer environment on non-Qubes
 
 * Ran by `run.sh`
+* Works with SecureDrop run in a docker container, see [SecureDrop docs](https://docs.securedrop.org/en/latest/development/setup_development.html) for setup instructions
+* Uses temporary configuration directories by default
+* Requests/responses to the Journalist API are sent directly via HTTP (and Tor is not used)
```

Comment:
nit: Tor also uses http, perhaps local webserver or webserver running in the local container would be clearer here?